libraries/mbed/hal/can_api.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     IRQ_PASSIVE,
     IRQ_ARB,
     IRQ_BUS,
-    IRQ_READY
+    IRQ_READY,
+    IRQ_RX1
```

Comment:
What CAN are you referring to ? stm32 CAN peripheral  module? Isn't this STM32 specific?